nano/core_test/toml.cpp

```diff
@@ -1038,3 +1038,47 @@ TEST (toml, log_config_no_required)
 
 	ASSERT_FALSE (toml.get_error ()) << toml.get_error ().get_message ();
 }
+
```

Comment:
I think in this case, a systest would had been a better option.
But you have done it now...